app/src/main/java/ch/epfl/polybazaar/chat/ChatMessageRecyclerAdapter.java

```diff
@@ -39,7 +39,7 @@ public int getItemViewType(int position) {
         Account user = AuthenticatorFactory.getDependency().getCurrentUser();
         assert(user != null);
 
-        if (message.getSender().equals(user.getEmail())) {
+        if (user == null || message.getSender().equals(user.getEmail())) {
```

Comment:
I understand this line is weird considering the assert above. But somehow it was needed to pass tests on Travis. I will probably delete the assertion